src/components/Explorer/ExplorerFormField.jsx

```diff
@@ -67,8 +80,8 @@ class ExplorerFormField extends React.Component {
             floatingLabelText={label}
             container="inline"
             autoOk
-            defaultDate={builder[builderField] ? new Date(builder[builderField]) : undefined}
-            onShow={this.resetField}
+            defaultDate={minDate ? new Date(new Date().setDate(new Date().getDate() - 30)) : undefined}
```

Comment:
There doesn't seem to be any way to remove the default date, perhaps we should look at setting this value a different way?

Reply:
e.g. if the user doesn't do anything, the value should be set to 30 days ago.

However, if they click on the field and don't select anything, it should clear the value.